services/admin.js

```diff
@@ -0,0 +1,44 @@
+const User = require('~/models/user')
+const Role = require('~/models/role')
+const {
+  roles: { ADMIN }
+} = require('~/consts/auth')
+const { USER_NOT_REGISTERED } = require('~/consts/errors')
+const { createError } = require('~/utils/errorsHelper')
+
+const adminService = {
+  getAdmins: async () => {
+    const role = await Role.findOne({ role: ADMIN }).lean().exec()
```

Comment:
you duplicate this query a lot of times. I guess will be better move this to role service or get from this service if it is already exists

Reply:
Role service was being implemented before and then it was decided to be removed, currently I don’t see a reason to create a whole service for just two cases of usage.